### PR TITLE
Add terms and definitions

### LINKED ISSUE #24: Cleanup terms and definitions
**Describe the bug**
Terms and definitions got merged in an intermediate state. Now the file needs to be cleaned up. Dee comments in https://github.com/asam-ev/OpenMATERIAL/pull/18

### PR DESCRIPTION
**Pull request by ClemensLinnhoff**
## Describe your changes
Adding terms and definitions as a result of multiple project meetings.

## Issue ticket number and link

Fixes #24, related to #18.

## Checklist before requesting a review

- [x] I have performed a self-review of my code/documentation.
- [x] My changes generate no new warnings during the documentation generation.

### CODE REVIEW

File: content/terms_and_definitions/terms_and_definitions.adoc
Comment (ACMackenthun) [inline content/terms_and_definitions/terms_and_definitions.adoc:54]: Proposal:

A visible and/or tangible thing with defined 3D boundaries and discrete meshes. An object ID can be assigned.

File: content/terms_and_definitions/terms_and_definitions.adoc
Comment (ACMackenthun) [inline content/terms_and_definitions/terms_and_definitions.adoc:77]:
```suggestion
Scenario::
```

File: content/terms_and_definitions/terms_and_definitions.adoc
Comment (ACMackenthun) [inline content/terms_and_definitions/terms_and_definitions.adoc:35]:
```suggestion
Standardized classification scheme that sorts materials into classes and groups of classes, for example, based on their physical properties.
```

File: content/terms_and_definitions/terms_and_definitions.adoc
Comment (mettelilienthal) [inline content/terms_and_definitions/terms_and_definitions.adoc:38]: ...in JSON format...

File: content/terms_and_definitions/terms_and_definitions.adoc
Comment (mettelilienthal) [inline content/terms_and_definitions/terms_and_definitions.adoc:21]: Simulation of multiple independent models running simultaneously and connected with interfaces.

File: content/terms_and_definitions/terms_and_definitions.adoc
Comment (mettelilienthal) [inline content/terms_and_definitions/terms_and_definitions.adoc:22]: the individual models.

File: content/terms_and_definitions/terms_and_definitions.adoc
Comment (mettelilienthal) [inline content/terms_and_definitions/terms_and_definitions.adoc:25]: Proposal:
Defines true static and dynamic states of objects. The true states are either determined by the content creator or defined by an omniscient observer (for example, simulation platform).

File: content/terms_and_definitions/terms_and_definitions.adoc
Comment (ClemensLinnhoff) [reply content/terms_and_definitions/terms_and_definitions.adoc:54]: "visible" is a bit fuzzy, especially if we are talking about multiple sensor technologies. Is a flat glass pane visible?

I like tangible, though! So if you could remove visible from your proposal, I would approve.

File: content/terms_and_definitions/terms_and_definitions.adoc
Comment (ClemensLinnhoff) [reply content/terms_and_definitions/terms_and_definitions.adoc:49]: Great catch. I like the differentiation.

File: content/terms_and_definitions/terms_and_definitions.adoc
Comment (ClemensLinnhoff) [reply content/terms_and_definitions/terms_and_definitions.adoc:38]: :+1:

File: content/terms_and_definitions/terms_and_definitions.adoc
Comment (ClemensLinnhoff) [reply content/terms_and_definitions/terms_and_definitions.adoc:21]: The models are not necessarily running simultaneously. In many cases they run sequentially every time step. So I would remove "running simultaneously".

File: content/terms_and_definitions/terms_and_definitions.adoc
Comment (ClemensLinnhoff) [reply content/terms_and_definitions/terms_and_definitions.adoc:41]: I would hold off the "geometric" for now. This is not decided yet. And I don't know if it is a good idea to name examples here. Texture could be a bit misleading and things like thickness are also not figured out, yet. Temperature is not really material property.

Coming up with these properties is kind of the core of this project. So I would not name them here in the definitions.

File: content/terms_and_definitions/terms_and_definitions.adoc
Comment (mettelilienthal) [inline content/terms_and_definitions/terms_and_definitions.adoc:13]: A mathematical coordinate-based representation of an object's geometrical surface.

File: content/terms_and_definitions/terms_and_definitions.adoc
Comment (mettelilienthal) [inline content/terms_and_definitions/terms_and_definitions.adoc:10]: The combination of an object's 3D geometry, assigned material properties, and specific metadata.

File: content/terms_and_definitions/terms_and_definitions.adoc
Comment (mettelilienthal) [inline content/terms_and_definitions/terms_and_definitions.adoc:18]: Proposal:
A 3D representation of a simulated sensor's surroundings. It can consist of multiple 3D assets.

File: content/terms_and_definitions/terms_and_definitions.adoc
Comment (mettelilienthal) [reply content/terms_and_definitions/terms_and_definitions.adoc:13]: General suggestion for 3D model/asset/environment: Maybe only list 3D model and then list asset and environment as related terms to better show how these concepts belong together?

File: content/terms_and_definitions/terms_and_definitions.adoc
Comment (mettelilienthal) [inline content/terms_and_definitions/terms_and_definitions.adoc:83]: Proposal: Rephrase the quote to an indirect one as to match the other definitions' template

The act of a signal traveling through the environement and potentially interacting with objects and particles.

File: content/terms_and_definitions/terms_and_definitions.adoc
Comment (ACMackenthun) [inline content/terms_and_definitions/terms_and_definitions.adoc:61]:
```suggestion
Class of objects with the same set of properties. Each instance of the class is described by the same properties, but has individual property values.
```

File: content/terms_and_definitions/terms_and_definitions.adoc
Comment (ClemensLinnhoff) [reply content/terms_and_definitions/terms_and_definitions.adoc:13]: Yes, referencing the definitions of asset and environment is a good idea!

File: content/terms_and_definitions/terms_and_definitions.adoc
Comment (ACMackenthun) [inline content/terms_and_definitions/terms_and_definitions.adoc:61]:
```suggestion
Class of objects with the same set of properties. Each instance of the class is described by the same properties, but has individual property values.
```

File: content/terms_and_definitions/terms_and_definitions.adoc
Comment (ACMackenthun) [inline content/terms_and_definitions/terms_and_definitions.adoc:56]: @ClemensLinnhoff: Not sure whether this term refers to a part/component of an object or a certain type of display feature. Let's discuss in our call this afternoon.

I have the impression that we need to define two concepts:
- Part/component of a 3D object: A signpost consists of two parts, a pole and plate. The pole is represented by a line, the plate is represented by a polygon.
- Material type/feature: Type of geometry used for display. Examples: line, polygon...

File: content/terms_and_definitions/terms_and_definitions.adoc
Comment (ACMackenthun) [inline content/terms_and_definitions/terms_and_definitions.adoc:46]: Proposal

Multispectral/hyperspectral imaging::
Method that captures detailed information across multiple wavelengths, allowing precise identification and analysis of materials.

File: content/terms_and_definitions/terms_and_definitions.adoc
Comment (ACMackenthun) [inline content/terms_and_definitions/terms_and_definitions.adoc:75]: Proposal

Post rendering effect::
Visual modification applied to graphics after initial rendering, enhancing appearance or realism without impairing performance.

File: content/terms_and_definitions/terms_and_definitions.adoc
Comment (mettelilienthal) [inline content/terms_and_definitions/terms_and_definitions.adoc:88]: Proposal:
Application that orchestrates and runs the simulations. It can be either an all-in-one solution including the generation of the 3D environment or a co-simulation platform orchestrating a co-simulation of multiple sensors, models, vehicle dynamics models, and so on.

File: content/terms_and_definitions/terms_and_definitions.adoc
Comment (ACMackenthun) [inline content/terms_and_definitions/terms_and_definitions.adoc:66]: I propose to define term "machine perception" instead of "perception", and to merge the direct citation into the definition. Do we need the part about traffic infrastructure in the general description?

Machine perception::
The ability of systems to interpret sensory data and to reliably detect objects and other road users relevant to the operation of automated driving. The data is used to generate situational awareness of the vehicle environment.

File: content/terms_and_definitions/terms_and_definitions.adoc
Comment (ACMackenthun) [inline content/terms_and_definitions/terms_and_definitions.adoc:32]:
```suggestion
The material IDs are a unique pointer to a set of material properties in separate material files.
```

File: content/terms_and_definitions/terms_and_definitions.adoc
Comment (ClemensLinnhoff) [reply content/terms_and_definitions/terms_and_definitions.adoc:66]: No I don't think we need the traffic infrastructure part.

File: content/terms_and_definitions/terms_and_definitions.adoc
Comment (ClemensLinnhoff) [reply content/terms_and_definitions/terms_and_definitions.adoc:70]: Color and opacity are kind of tied to to the visual spectrum. I would rather use permittivity, index of refraction, density as examples.

File: content/terms_and_definitions/terms_and_definitions.adoc
Comment (ClemensLinnhoff) [reply content/terms_and_definitions/terms_and_definitions.adoc:32]: Shouldn't it be "A material ID is a unique pointer ..." or "The material IDs are unique pointers ..."?

Comment (ACMackenthun) [reply content/terms_and_definitions/terms_and_definitions.adoc:32]: Yes, thanks for the remark. I updated the suggestion

File: content/terms_and_definitions/terms_and_definitions.adoc
Comment (mettelilienthal) [reply content/terms_and_definitions/terms_and_definitions.adoc:41]: zweiten Satz (vorerst) streichen

File: content/terms_and_definitions/terms_and_definitions.adoc
Comment (mettelilienthal) [reply content/terms_and_definitions/terms_and_definitions.adoc:46]: material definitions with various wavelengths? wide range of wavelengths
imaging => material

File: content/terms_and_definitions/terms_and_definitions.adoc
Comment (mettelilienthal) [reply content/terms_and_definitions/terms_and_definitions.adoc:56]: discuss Geometry later
Subtype will be discussed in vehicle hierarchy

File: content/terms_and_definitions/terms_and_definitions.adoc
Comment (mettelilienthal) [reply content/terms_and_definitions/terms_and_definitions.adoc:41]: are there any physical properties?

File: content/terms_and_definitions/terms_and_definitions.adoc
Comment (ACMackenthun) [reply content/terms_and_definitions/terms_and_definitions.adoc:10]: done

File: content/terms_and_definitions/terms_and_definitions.adoc
Comment (ACMackenthun) [reply content/terms_and_definitions/terms_and_definitions.adoc:13]: updated definition

TODO: add relations

File: content/terms_and_definitions/terms_and_definitions.adoc
Comment (ACMackenthun) [reply content/terms_and_definitions/terms_and_definitions.adoc:18]: done

File: content/terms_and_definitions/terms_and_definitions.adoc
Comment (ACMackenthun) [reply content/terms_and_definitions/terms_and_definitions.adoc:21]: done

File: content/terms_and_definitions/terms_and_definitions.adoc
Comment (ACMackenthun) [reply content/terms_and_definitions/terms_and_definitions.adoc:22]: done

File: content/terms_and_definitions/terms_and_definitions.adoc
Comment (ACMackenthun) [reply content/terms_and_definitions/terms_and_definitions.adoc:25]: done

File: content/terms_and_definitions/terms_and_definitions.adoc
Comment (ACMackenthun) [reply content/terms_and_definitions/terms_and_definitions.adoc:32]: done

File: content/terms_and_definitions/terms_and_definitions.adoc
Comment (ACMackenthun) [reply content/terms_and_definitions/terms_and_definitions.adoc:38]: done

File: content/terms_and_definitions/terms_and_definitions.adoc
Comment (ACMackenthun) [reply content/terms_and_definitions/terms_and_definitions.adoc:41]: done

File: content/terms_and_definitions/terms_and_definitions.adoc
Comment (ACMackenthun) [reply content/terms_and_definitions/terms_and_definitions.adoc:46]: changed to "Material type that interacts with light across multiple wavelengths."

File: content/terms_and_definitions/terms_and_definitions.adoc
Comment (ACMackenthun) [reply content/terms_and_definitions/terms_and_definitions.adoc:49]: done, added terms "surface normal" and "vertex normal" to glossary

File: content/terms_and_definitions/terms_and_definitions.adoc
Comment (ACMackenthun) [reply content/terms_and_definitions/terms_and_definitions.adoc:54]: done, updated description and removed "visible"

File: content/terms_and_definitions/terms_and_definitions.adoc
Comment (ACMackenthun) [reply content/terms_and_definitions/terms_and_definitions.adoc:66]: done

File: content/terms_and_definitions/terms_and_definitions.adoc
Comment (ACMackenthun) [reply content/terms_and_definitions/terms_and_definitions.adoc:70]: done

File: content/terms_and_definitions/terms_and_definitions.adoc
Comment (ACMackenthun) [reply content/terms_and_definitions/terms_and_definitions.adoc:75]: done

File: content/terms_and_definitions/terms_and_definitions.adoc
Comment (ACMackenthun) [reply content/terms_and_definitions/terms_and_definitions.adoc:83]: done

File: content/terms_and_definitions/terms_and_definitions.adoc
Comment (ACMackenthun) [reply content/terms_and_definitions/terms_and_definitions.adoc:88]: done

File: content/terms_and_definitions/terms_and_definitions.adoc
Comment (ACMackenthun) [reply content/terms_and_definitions/terms_and_definitions.adoc:91]: done

File: content/terms_and_definitions/terms_and_definitions.adoc
Comment (ACMackenthun) [reply content/terms_and_definitions/terms_and_definitions.adoc:56]: replaced with:

Object part::
A segment of an object that has distinct characteristics, for example, material, shape, or function.

Example: A car door differs from the rest of the vehicle due to its function to open and close.

**Review by ACMackenthun — APPROVED** [review]

**Comment by ClemensLinnhoff** [comment]
@ACMackenthun please fix the DCO of your last commit.

File: content/terms_and_definitions/terms_and_definitions.adoc
Comment (ClemensLinnhoff) [inline content/terms_and_definitions/terms_and_definitions.adoc:59]: The term light commonly only refers to the visible spectrum. I would rather say
"Material with wavelength dependent properties."

File: content/terms_and_definitions/terms_and_definitions.adoc
Comment (ClemensLinnhoff) [inline content/terms_and_definitions/terms_and_definitions.adoc:103]:
```suggestion
co-simulation platform orchestrating a co-simulation of multiple sensor models, vehicle dynamics models, and so on.
```

**Review by FKlopfer — APPROVED** [review]

File: content/terms_and_definitions/terms_and_definitions.adoc
Comment (mettelilienthal) [inline content/terms_and_definitions/terms_and_definitions.adoc:60]:
```suggestion
Material with wavelength-dependent properties. 

Multispectral material shows up in imagery as 3-10 wider bands.
Hyperspectral material shows hundreds of narrow bands. 
```

File: content/terms_and_definitions/terms_and_definitions.adoc
Comment (LudwigFriedmann) [inline content/terms_and_definitions/terms_and_definitions.adoc:45]: A mapping of material IDs to parts of a of 3D models or pixels of textures mapped to these parts.

File: content/terms_and_definitions/terms_and_definitions.adoc
Comment (LudwigFriedmann) [inline content/terms_and_definitions/terms_and_definitions.adoc:57]: Denotes a physical, chemical, or geometric property of a material.

File: content/terms_and_definitions/terms_and_definitions.adoc
Comment (LudwigFriedmann) [inline content/terms_and_definitions/terms_and_definitions.adoc:60]: Material with wavelength-specific properties that is defined for multiple spectra.

File: content/terms_and_definitions/terms_and_definitions.adoc
Comment (LudwigFriedmann) [inline content/terms_and_definitions/terms_and_definitions.adoc:81]: [...] of an object or material [...]

**Review by LudwigFriedmann — CHANGES_REQUESTED** [review]
Added some minor corrections

**Review by AsamDiegoSanchez — APPROVED** [review]
From my point of view it can be merged after adding the last suggested changes.

File: content/terms_and_definitions/terms_and_definitions.adoc
Comment (mettelilienthal) [inline content/terms_and_definitions/terms_and_definitions.adoc:15]:
```suggestion
surface. Material properties used for rendering purposes can be part of the 3D model. This excludes physical material properties. For a combination of 3D
```

File: content/terms_and_definitions/terms_and_definitions.adoc
Comment (mettelilienthal) [inline content/terms_and_definitions/terms_and_definitions.adoc:26]:
```suggestion
co-simulation platform handles the timing, synchronization, and data transfer between the
```

File: content/terms_and_definitions/terms_and_definitions.adoc
Comment (mettelilienthal) [inline content/terms_and_definitions/terms_and_definitions.adoc:45]:
```suggestion
A mapping of material IDs to meshes of 3D models, or pixels of textures mapped to these meshes. The material ID is a unique
```

File: content/terms_and_definitions/terms_and_definitions.adoc
Comment (mettelilienthal) [inline content/terms_and_definitions/terms_and_definitions.adoc:57]:
```suggestion
Denotes a physical, chemical, or geometric property of a material.
```

File: content/terms_and_definitions/terms_and_definitions.adoc
Comment (mettelilienthal) [inline content/terms_and_definitions/terms_and_definitions.adoc:60]:
```suggestion
Material with wavelength-specific properties that are defined for multiple spectra. 
```

File: content/terms_and_definitions/terms_and_definitions.adoc
Comment (mettelilienthal) [inline content/terms_and_definitions/terms_and_definitions.adoc:81]:
```suggestion
Denotes a characteristic of an object or material that can be measured.
```

**Comment by mettelilienthal** [comment]
@LudwigFriedmann I have updated the file with your desired changes. If you agree with everything now and approve the PR, we can merge it into main.

**Review by LudwigFriedmann — APPROVED** [review]

**Comment by FKlopfer** [comment]
@AsamDiegoSanchez , can you merge the PR?

**Pull request merged**
